project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -1744,6 +1814,19 @@ function c4m_message_format_help_section_url(Message $message) {
 }
 
 /**
+ * Returns site contact link token.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *   Site contact link.
+ */
+function c4m_message_format_site_contact_link(Message $message) {
+  return 'mailto:capacity4dev@ec.europa.eu';
```

Comment:
@soniCaH 
can we use `variable_get('site_mail');` in here ?
or should we add another variable, something like `c4m_notifications_site_contant_mail`